### PR TITLE
Add feature authorization deposit

### PR DESCRIPTION
**Pull request by AngieCristina**
Close #42 
Pair programming with Paul Portillo
- Create deposit controller, deposit rol, deposit routes, deposit policy.

### CODE REVIEW

**Comment by AngieCristina** [comment]
Close #42

File: app/policies/deposit_policy.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  attr_reader :user, :deposit
+
+  def initialize (user, deposit)
+    @user = user
+    @deposit = deposit
+  end
+  
+
+  def index?
+    user.deposit? || user.admin?
```
Comment (Linzeur) [inline app/policies/deposit_policy.rb]: You considered it, good!!

File: db/seeds.rb [hunk 2]
```diff
@@ -62,8 +62,7 @@
   city: "Lima",     
   country: "Peru",
   address: "Miraflores",  
-  role: "admin",
-  authentication_token: Devise.friendly_token[0, 30]     
+  role: "admin"
```
Comment (Linzeur) [inline db/seeds.rb]: Thanks for fix that!

File: db/seeds.rb [hunk 3]
```diff
@@ -113,4 +112,5 @@
   freight_value: 200,
   user_id: usermailer.id,
   sender_id: sendermailer.id
-)
\ No newline at end of file
+)
+p "Seed complete"
```
Comment (Linzeur) [inline db/seeds.rb]: Thanks for add a message of confirmation

Comment (yummta) [inline db/seeds.rb]: Thanks for this detail.

**Pull request merged**